include/fastq_conversion.hpp

```diff
@@ -0,0 +1,14 @@
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/io/sequence_file/all.hpp>
+#include <seqan3/io/sequence_file/input.hpp>
+#include <seqan3/io/sequence_file/output.hpp>
+
+void convert_fastq(std::filesystem::path fastq_file, std::filesystem::path out)
+{
+    seqan3::sequence_file_input fin{fastq_file};
+    seqan3::sequence_file_output fout{std::cout, seqan3::format_fasta{}};
+    if (out != "") // If an output path is given
+        seqan3::sequence_file_output fout{out/"output.fasta"};
```

Comment:
since you used `std::filesystem::path` is is already the path including the output file name.
```suggestion
        seqan3::sequence_file_output fout{out};
```